lib/ae_mdw/channels.ex

```diff
@@ -131,7 +133,13 @@ defmodule AeMdw.Channels do
 
   defp build_streamer(state, scope, cursor) do
     fn direction ->
-      Collection.stream(state, @table_activation, direction, scope, cursor)
+      [@table_active_activation, @table_inactive_activation]
```

Comment:
what you think about using the inverse order? when it goes on the backwards default direction it would render the active ones firstly (maybe it's helpful on heights with many inactive ones).

Reply:
It already returns results in backwards activation order. So the channels that were activated last will show up first by default

Reply:
it shows the inactive firstly (on backwards) because of this order: `[@table_active_activation, @table_inactive_activation]`

Reply:
`Collection.merge/2` merges results in the right order, it grabs the first key from both tables and then merges them accordingly, iteratively. I think what you're thinking of is `Stream.concat/2`

Reply:
Since the table index is `{activation_height, channel_pk}`, when on equal heights, the first result shown is always the one following the `channel_pk` order:
```
ch1 = Validate.id!("ch_2dE8Co8wHju7qb46ezjBwUpvDr6gnQxr2vEjxpWwEsFC4jBurr")
ch2 = Validate.id!("ch_NAbNxGCpXSAzTGoXxKPnEmeWLzKYT5oN5fQqD6x1b4DQ77J5C")

ch2 < ch1
```
which is why on default order `ch_2dE8Co8wHju7qb46ezjBwUpvDr6gnQxr2vEjxpWwEsFC4jBurr` is shown first

Reply:
Okay the `Collection.merge` doesn't preserve the order then. I will leave it to the final users if it's an important request.